api/opds_for_distributors.py

```diff
@@ -357,7 +357,10 @@ def update_work_for_edition(self, *args, **kwargs):
         pool, work = super().update_work_for_edition(
             *args, is_open_access=False, **kwargs
         )
-        pool.unlimited_access = True
+
+        if pool:
```

Comment:
One thing I'm not sure about: does it make sense for the pool to be returned as a None from super().update_work_for_edition.  The `update_work_for_edition` seems to allow for it  and downstream code seems to expect it, but I'm wondering if makes sense in this context.

Reply:
It does seem like the super class expects to return None in some cases. So I think this is an okay fix.